src/js/Setup.js

```diff
@@ -356,6 +356,61 @@ function createCindyNow() {
     if (typeof scriptconf === "string" && scriptconf.search(/\*/) >= 0) scriptpat = scriptconf;
     if (typeof scriptconf !== "object") scriptconf = null;
 
+    /*
+Loads CindyScript files (marked by .cjs file ending) and adds their code AT THE BEGINNING of the init script.
+The file names (and paths) have to be listed in the dictionary that's passed to CindyJS/createCindy as an array for the key 'import'. I.e., like this:
+          var cdy = CindyJS({
+                ports: [{ id: "CSCanvas", width: 500, height: 500 }],
+                scripts: "cs*",
+                import: ["cindyscript_libraries/libraryA", "cindyscript_libraries/libraryB"]
+
+            });
+Note that the libraries get prepended to the init-script BACK TO FRONT. So, in the example above, the code from libraryA comes before the code from libraryB whoch comes before the custom user code. That way, libraries can reference each others code.
```

Comment:
This cautionary message is a bit confusing – they are prepended back-to-front, yes, but actually they just come in the order in which they are (as expected). So you could also write "The libraries are prepended as a whole in the order listed, so libraryA is loaded before libraryB."

Reply:
Oh, that's much better, yes!